js/tree.js

```diff
@@ -372,11 +503,36 @@
 	};
 
 	$.fn.tree.defaults = {
-		dataSource: function (options, callback) {},
+		dataSource: function dataSource(options, callback) {},
 		multiSelect: false,
 		cacheItems: true,
 		folderSelect: true,
-		itemSelect: true
+		itemSelect: true,
+		/*
+		* currently calling "open" again on a folder will close it. Setting
+		* `ignoreRedundantOpens` to `true` will make the folder instead ignore
+		* the redundant call and stay open.
+		*/
+		ignoreRedundantOpens: false,
```

Comment:
Makes sense. Can you just add a bit more detail to the explanation? Maybe even point to a release or a github issue where this will change?